src/test/java/org/broadinstitute/hellbender/tools/walkers/qc/CheckPileupIntegrationTest.java

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.hellbender.tools.walkers.qc;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * Run validating pileup across a set of core data as proof of the integrity of the GATK core.
+ *
+ * Tests both types of old-school pileup formats (basic and consensus).
+ *
+ * @author Daniel Gómez-Sánchez (magicDGS)
+ */
+public class CheckPileupIntegrationTest extends CommandLineProgramTest {
+    private static final String TEST_DATA_DIRECTORY = publicTestDir + "org/broadinstitute/hellbender/engine/";
+    private static final String TEST_OUTPUT_DIRECTORY = publicTestDir + "org/broadinstitute/hellbender/tools/walkers/qc/pileup/";
+
+    /**
+     * This test runs on a consensus pileup containing 10-column lines for SNPs and 13-column lines for indels
+     */
+    @Test(enabled = false)
+    public void testConsensusPileup() throws IOException {
+        // TODO
+    }
+
+    /**
+     * This test runs on a basic pileup obtained with samtools (version 1.1) and options -B --min-BQ 0
```

Comment:
would you mind using a more recent samtools to regenerate the test files and use mpileup?